contracts/interfaces/Compound/ICToken.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity >=0.7.2;
+pragma experimental ABIEncoderV2;
+
+/**
+ * @title Compound's CToken Contract
+ * @notice Abstract base for CTokens
+ * @author Compound
+ */
+interface ICToken {
```

Comment:
I wonder if we can import this interface from a library like OZ or Compound

Reply:
Nothing like this in compound library